examples/energy-management-app/energy-management-common/src/DEMTestEventTriggers.cpp

```diff
@@ -0,0 +1,401 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <DeviceEnergyManagementDelegateImpl.h>
+#include <EVSEManufacturerImpl.h>
+#include <app/clusters/device-energy-management-server/DeviceEnergyManagementTestEventTriggerHandler.h>
+
+#include <EnergyTimeUtils.h>
+
+#include "FakeReadings.h"
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::DeviceEnergyManagement;
+
+#define MAX_SLOTS 10
```

Comment:
why the redefine?

Prefer C++-style constexpr constants throughout.

Reply:
Done